tests/eibc_fulfillment_test.go

```diff
@@ -23,6 +24,422 @@ import (
 	"github.com/decentrio/rollup-e2e-testing/testutil"
 )
 
+func TestEIBCFulfillment_on_one(t *testing.T) {
```

Comment:
We should update the name of func like `TestEIBCFulfillOnOneRollApp`, and add a comment to explain the case of testing